packages/cli/src/push/index.ts

```diff
@@ -85,7 +85,7 @@ export class Push extends TypedEmitter<PushEvents> {
 		this.backend.sendToAll(type, data);
 	}
 
-	send<Type extends PushType>(type: Type, data: PushPayload<Type>, pushRef: string) {
+	send({ type, data }: PushMessage, pushRef: string) {
```

Comment:
didn't we use to have it this way, and then we split it up into args?

Reply:
They've been split ever since the introduction of DI for push: https://github.com/n8n-io/n8n/pull/5509/files#diff-b26a42aa0df202084a61c88a206f1e0032f64a3379845786a493264a1ae6d264R32

Reply:
type inference is definitely easier this way. not sure why I made the switch back then. 🤷🏽